include/tvm/relay/type.h

```diff
@@ -277,6 +277,38 @@ class TypeReporterNode : public Node {
    *      true if solver can still proceed.
    */
   TVM_DLL virtual bool AssertEQ(const IndexExpr& lhs, const IndexExpr& rhs) = 0;
+  /*!
+   * \brief assert shape expression less than other.
+   * \param lhs The left operand.
+   * \param rhs The right operand.
```

Comment:
Just reduce it is Assert(const IndexExpr& cond), and you can check if cond is zero